### PR TITLE
GIX-1353: Poll accounts

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Add a service to retry getting accounts if they fail.

# Changes

* New accounts service `pollAccounts`.

# Tests

* Test new accounts service `pollAccounts`.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@mstrasinskis @dskloetd please review

File: frontend/src/lib/services/accounts.services.ts [hunk 1]
```diff
@@ -317,3 +322,53 @@ const renameError = ({
 
   return { success: false, err: labelKey };
 };
+
+const pollLoadAccounts = async (params: {
+  identity: Identity;
+  certified: boolean;
+}): Promise<AccountsStoreData> =>
+  poll({
+    fn: () => loadAccounts(params),
```
Comment (mstrasinskis) [inline frontend/src/lib/services/accounts.services.ts:331]: Would be great to retry per account. Not a strong opinion and definitely not for this PR.

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

File: frontend/src/lib/services/accounts.services.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/services/accounts.services.ts:331]: Yes, I agree. But I didn't want to go into all the details on where exactly to retry or not...

It won't add much value for now anyway. All these calls are in the NNS subnet. If that's overloaded, any of these calls would need to be retried.

**Pull request merged**